FluentTerminal.Models/ShellProfile.cs

```diff
@@ -64,23 +64,29 @@ public string TranslateLineEndings(string content)
         public Guid TerminalThemeId { get; set; }
         public ICollection<KeyBinding> KeyBindings { get; set; } = new List<KeyBinding>();
 
-        public override bool Equals(object obj)
+        public virtual bool EqualTo(ShellProfile other)

```

Comment:
what was the problem with .Equals()?

Reply:
it is not recommended to do it partially. If you want to override `Equals()`, you should always override `GetHashCode()`. Only this way `Equals()` will fully remain the expected behavior.